public/app/features/dashboard/utils/loadSnapshotData.ts

```diff
@@ -0,0 +1,27 @@
+import { applyFieldOverrides, DefaultTimeRange, LoadingState, PanelData } from '@grafana/data';
+import { config } from 'app/core/config';
+import { getDatasourceSrv } from 'app/features/plugins/datasource_srv';
+import { DashboardModel, PanelModel } from '../state';
+import { getProcessedDataFrames } from '../state/runRequest';
+
+export function loadSnapshotData(panel: PanelModel, dashboard: DashboardModel): PanelData {
+  const data = getProcessedDataFrames(panel.snapshotData);
+
+  return {
+    timeRange: DefaultTimeRange,
+    state: LoadingState.Done,
+    series: applyFieldOverrides({
+      data,
+      fieldConfig: {
+        defaults: {},
+        overrides: [],
+      },
```

Comment:
Won't this break data links if they are stored in the snapshot?

Reply:
should already be on the data frame stored in the panel as the stores snapshot data already has gone through this